ppx/melange_ppx.ml

```diff
@@ -697,6 +697,71 @@ module Mapper = struct
             str
         | _ -> super#structure_item str
 
+      method! structure str =
+        let extract_mel_as_ident ~loc x =
+          match x with
+          | PStr [ { pstr_desc = Pstr_eval ({ pexp_desc; _ }, _); _ } ] -> (
+              match pexp_desc with
+              | Pexp_constant (Pconst_string (name, _, _))
+              | Pexp_construct ({ txt = Lident name; _ }, _)
+              | Pexp_ident { txt = Lident name; _ } ->
+                  name
```

Comment:
Should `Ext_ident.convert` be called somewhere? I think it's still useful to prevent users from having ids with reserved words in JS.

Reply:
That should already work later on in the compiler backend. I'll add a test for it.

Reply:
tested in 6d5855b90